cli/tests/environment-activate.bats

```diff
@@ -413,3 +413,23 @@ env_is_activated() {
   assert_line --partial "hello is $(realpath $PROJECT_DIR)/.flox/run/"
   assert_line "baz"
 }
+
+# ---------------------------------------------------------------------------- #
+
+# bats test_tags=activate,activate:inplace-reactivate
+@test "'flox activate' only patches PATH when already activated" {
+  run "$FLOX_BIN" activate -- "$FLOX_BIN" activate --in-place
+  assert_success
+  if [[ -e  /usr/libexec/path_helper ]]; then
+    assert_output --regexp "^(export PATH=.+)$"
+  else
+    assert_output --regexp "^true$"
```

Comment:
Why do we expect to see `true` here?
There isn't sufficient context in this test case to understand the behavior you're auditing.

Reply:
the output of that command is used for in-place activation using `eval "$(flox activate [--in-place]`).
This application of flox activate **ignores whether the env is already active**.
On macos we need to fix what the path_helper breaks, on linux we have nothing to do (yet?).

I printed `true` to have a guaranteed `0` exit code.
however `eval ""` is valid too and maybe less confusing than printing "true"...

Reply:
Added some inline comments in the test and changed  `true` -> `ε`